public/stylesheets/site/2.0/02-elements.css

```diff
@@ -182,7 +182,7 @@ sup {
   line-height: 0;
 }
 
-/*http://otwcode.github.com/docs/front_end_coding/accessible-tables*/
+/*https://otwcode.github.io/docs/front_end_coding/coding-accessible-tables*/
```

Comment:
It looks like this is giving a 404 and it should be just `https://otwcode.github.io/docs/front_end_coding/accessible-tables`